marklogic-data-hub-central/build.gradle

```diff
@@ -186,6 +188,63 @@ test {
 	useJUnitPlatform()
 }
 
+ospackage {
```

Comment:
Can you try moving this config to "build-rpm.gradle" in the same directory? I've found that build.gradle files can become very long over time - this seems like an easy way to split it up. On the other hand, sometimes extracting a separate file is harder than it should be.

Ideally, we can move the ospackage plugin declaration to build-rpm.gradle, and also the entire ospackage block. The trick is often where you apply the separate file within build.gradle. We may still need ospackage declared in build.gradle. 

The benefit would be that we'd have one separate Gradle file for everything pertaining to the rpm, instead of having to sift through hundreds of lines of config in build.gradle to find it.

Reply:
Done